arch/sim/include/inttypes.h

```diff
@@ -44,7 +44,7 @@
  * Pre-processor Definitions
  ****************************************************************************/
 
-#if defined(__APPLE__)
+#if defined(CONFIG_HOST_MACOS)
```

Comment:
why not use __APPLE__?

Reply:
because of https://github.com/apache/incubator-nuttx/commit/b82756539623c2561c0ab612b2089e111399cbd0